torch_dag/core/unstructured_to_structured.py

```diff
@@ -279,7 +279,7 @@ def build_dag_by_tracing(self) -> dag_module.DagModule:
                             predecessors=predecessors,
                         )
 
-                    if node.next.op == 'output':
+                    if 'output' in [node.name for node in list(node.users.keys())]:
```

Comment:
This I believe can be slightly simplified:
 
 `if 'output' in [node.name for node in node.users]:`

Reply:
you are right, thanks